libs/community/langchain_community/document_loaders/html.py

```diff
@@ -27,6 +27,35 @@ class UnstructuredHTMLLoader(UnstructuredFileLoader):
     https://unstructured-io.github.io/unstructured/bricks.html#partition-html
     """
 
+    def __init__(
+        self,
+        file_path: Union[str, List[str]] = None,
+        **kwargs: Any,
+    ):
+        def _url_to_file(url: str) -> str:
+            import tempfile
+
+            import requests
+
+            r = requests.get(url)
```

Comment:
we should try to avoid making requests in initialization, could we add this functionality to `load`?